tests/formulagrader/test_formulagrader.py

```diff
@@ -446,10 +446,18 @@ def is_coterminal_and_large(comparer_params, student_input, utils):
         reduced = student_input % (360)
         return utils.within_tolerance(answer, reduced) and student_input > min_value
 
-    mocked = mock.Mock(side_effect=is_coterminal_and_large,
-                # The next two kwargs ensure that the Mock behaves nicely for inspect.getargspec
-                spec=is_coterminal_and_large,
-                func_code=is_coterminal_and_large.func_code,)
+    mocked = (mock.create_autospec(is_coterminal_and_large,
+                                   side_effect=is_coterminal_and_large)
+              if six.PY3 else
```

Comment:
Yes, this works, but I'd much prefer a normal if statement here for readability.